### PR TITLE
Medienkompetenz again

### PR DESCRIPTION
**Pull request by edgalindo**

### CODE REVIEW

File: source/projects/medienkompetenzen/de/index.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+**Leistungen für Migranti*nnen**
+* Aufenthaltsgenehmigung
+* Arbeitsgenehmigung für Student\*innen
+
+**Organisatorische Dienstleistungen**
+* E-Mail Einrichten
+* Arzttermine online machen
+
+![Digital- Medienkompetenzen in der Bibliothek](/projects/medienkompetenzen/images/thumbnail.jpg)
```
Comment (fabianmoronzirfas) [inline source/projects/medienkompetenzen/de/index.md:74]: I think this should not be here.

Comment (fabianmoronzirfas) [reply source/projects/medienkompetenzen/de/index.md:74]: @edgalindo You have the markdown img link and the include

Comment (edgalindo) [reply source/projects/medienkompetenzen/de/index.md:74]: if I use following {% include macro-image-section-markdown.html src="/projects/medienkompetenzen/images/thumbnail.jpg" caption="Digital- Medienkompetenzen in der Bibliothek" %}, the image size is too big for the area

Comment (fabianmoronzirfas) [reply source/projects/medienkompetenzen/de/index.md:74]: Means the image is to small and gets pumped up to fit the column I guess.
 You meved back to having it included using `alt text`

One thing you _could_ do is embed it into a non rectangular frame with a size larger then the column.

**Pull request merged**